spec/peep_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'peep'
+
+describe Peep do
+  describe '.all' do
+    it 'Returns all peeps in database' do
```

Comment:
perhaps a second expect line checking the length is one once a peep is added would cement this function is working, just because if it wasn't working the length would be 0 also?

Reply:
just seen the test people, so perhaps the .all test can be removed because that feature is exhibited below